proposal/crystal0523/README.md

```diff
@@ -0,0 +1,47 @@
+# Fourier Transform for Image Processing
+## Basic Information
+* GitHub repository: https://github.com/crystal0523/nsdhw_21au
```

Comment:
This is a fork of the homework repository and cannot be used for your project repository.  Please create a new repository dedicated for your project.

Reply:
Got it, thanks for reminding me.